app/services/document.js

```diff
@@ -0,0 +1 @@
+export { default } from "ember-alexandria/services/document";
```

Comment:
You can delete this file.

Reply:
I cannot. Otherwise, the tests will fail.

Reply:
You need this in all engine tests to tell ember where to find the imports:
```js
import engineResolverFor from "ember-engines/test-support/engine-resolver-for";

const modulePrefix = "ember-alexandria";
const resolver = engineResolverFor(modulePrefix);

module("Integration | Component | document-card", function (hooks) {
  setupRenderingTest(hooks, { resolver });
```

Reply:
Then you can delete the export

Reply:
This causes problem with Mirage.

I had to introduce a custom resolver to fix this in the integration tests. I would rather not introduce this here as-well. Is there a reason, we cannot export this service?

Reply:
If its a pain then just leave the export.